components/odh-notebook-controller/config/manager/manager.yaml

```diff
@@ -6,6 +6,10 @@ metadata:
   namespace: system
 spec:
   replicas: 1
+  strategy:
```

Comment:
The same for on https://github.com/opendatahub-io/kubeflow/blob/v1.9-branch/components/odh-notebook-controller/config/manager/manager.yaml#L9-L12